README.md

```diff
@@ -110,7 +121,17 @@ plt.show()
 
 ## Contributing
 
-To contribute to *pydicom*, read our [contribution guide](https://github.com/pydicom/pydicom/blob/master/CONTRIBUTING.md).
+We are all volunteers working on *pydicom* in our free time. As our 
+resources are limited, we very much value your contributions, be it bug fixes, new 
+core features, or documentation improvements. For more information, please
+read our [contribution guide](https://github.com/pydicom/pydicom/blob/master/CONTRIBUTING.md).
 
-To contribute an example or extension of *pydicom* that doesn't belong with the core software, see our contribution repository:
+If you have examples or extensions of *pydicom* that don't belong with the 
+core software, but that you deem useful to others, you can add them to our 
+contribution repository:
 [contrib-pydicom](https://www.github.com/pydicom/contrib-pydicom).
+
+If you have a software package based on *pydicom* that you want to be more 
+tightly integrated into the *pydicom* ecosystem, you may consider to apply
+for transferring it into the *pydicom* organization. You can do so by creating 
+a general discussion item with a short description of the module.
```

Comment:
I'm not sure about this - I also wanted to document the basic reqiurements for such a transfer somewhere else, but first wanted to know if we want to add this at all.

Reply:
I am also not sure about this - I think by adding projects to the pydicom organization, we are essentially endorsing certain libraries over others.  The only benefit is to users, in helping them find libraries they might not otherwise have known about.  

I'd suggest we leave this out for now, and leave it to the pydicom owners to offer a move to the organization, in cases where a clear 'winner' exists that people should know about, that doesn't already have a strong management team under its own organization / individual account.